src/Arrays/MPIStateArrays.jl

```diff
@@ -784,6 +784,46 @@ end
 
 @init tictoc()
 
-include("MPIStateArrays_kernels.jl")
+using Requires
+@init @require CUDAnative = "be33ccc6-a3ff-5ff2-a52e-74243cff1e17" begin
+    using .CUDAnative
```

Comment:
Do we still need this?

Reply:
Not sure. Will try to remove and see.

Reply:
Fine to remove will fix up the commits and push this out.